backend_python/backend/game.py

```diff
@@ -1,21 +1,20 @@
-try:
-    from backend_python.backend.town import Town
-except ModuleNotFoundError:
-    from town import Town
-
+from backend.town import Town
 import code
 import logging
 from datetime import datetime
-
 from flask import jsonify
 from transitions import Machine, State, Transition
 
-logging.basicConfig(filename='state_machine.log', level=logging.DEBUG)
-logging.getLogger('transitions').setLevel(logging.DEBUG)
+log_handler = logging.FileHandler('log/state_machine.log')
+log_handler.setLevel(logging.DEBUG)
+log_handler.setFormatter(logging.Formatter('%(asctime)-15s %(message)s'))
 
+logger = logging.getLogger('transitions')
+logger.setLevel(logging.DEBUG)
+logger.handlers = [log_handler]
 
 class Game(object):
-    
```

Comment:
@AmadeuszFilipek u sure u have https://github.com/editorconfig/editorconfig-vscode added to VSCode? Seems like it does not work, because vim on save is removing those trailing spaces.